abis/0.8.19/GovernorOLAS.json

```diff
@@ -1,1425 +0,0 @@
-{
```

Comment:
This ABI is not used anywhere in contracts or setup, it was the intermediary one